packages/SwingSet/src/kernel/vatManager/supervisor-subprocess-xsnap.js

```diff
@@ -238,20 +249,23 @@ function makeWorker(port) {
         assert(!dispatch, 'cannot setBundle again');
         const enableDisavow = !!args[4];
         const enableVatstore = !!args[5];
-        const gcEveryCrank = args[6] === undefined ? true : !!args[6];
+        const consensusMode = !!args[6];
+        const gcEveryCrank = args[7] === undefined ? true : !!args[7];
         return setBundle(
           args[0],
           args[1],
           args[2],
           args[3],
           enableDisavow,
           enableVatstore,
+          consensusMode,
```

Comment:
If we already have to send a `consensusMode` flag with every delivery (and I agree we do, to switch out of it after a snapshot restore), do we also need it at `setBundle` time? I can imagine it controlling behavior other than logging, but snapshot-restore means whatever that extra behavior is, it won't actually get enabled, right?

Reply:
It needs to be in `setBundle` since evaluated module-level code could do the same kinds of shenanigans as any other user-level code.